### PR TITLE
INTEGRATIONS: Retrieve AI Model By Name

### LINKED ISSUE #298: INTEGRATIONS: Retrieve AI Model By Name
# The Ask
Develop an Integration test to verify that the Retrieve AI Model By Name component is working correctly

# How to Complete this Task
Here's some steps to complete this deliverable.
- Watch how The Standard Community developed a similar Integration Tests for OpenAI [here](https://www.youtube.com/watch?v=C-fzlF2k7SM)
- You cannot start working on this issue until #296  is completed and closed

# How to claim this issue?
- Post on this issue a comment that says: "I claim this issue"
- You will be given 24 hours to complete the issue - if no PR is opened within 24 hours someone else will claim it
- If you have an open PR but is inactive for more than 24 hours from the last review comment we will shutdown your PR

### PR DESCRIPTION
**Pull request by Gummimundur**
Closes #298

### CODE REVIEW

**Comment by Gummimundur** [comment]
image

File: Standard.AI.OpenAI.Tests.Integration/APIs/AIModels/AIModelsApiTests.cs [hunk 1]
```diff
@@ -16,8 +16,8 @@ public AIModelsApiTests()
         {
             var openAIConfigurations = new OpenAIConfigurations
             {
-                ApiKey = Environment.GetEnvironmentVariable("ApiKey"),
-                OrganizationId = Environment.GetEnvironmentVariable("OrgId"),
+                ApiKey = Environment.GetEnvironmentVariable("OpenAIApiKey"),
```
Comment (hassanhabib) [inline Standard.AI.OpenAI.Tests.Integration/APIs/AIModels/AIModelsApiTests.cs:19]: This change here will break the integration tests (if enabled).
These names are set on this repo as env variables. I'm okay with changnig the names I think they are good. but It shouldn't be a part of this PR.

Comment (hassanhabib) [reply Standard.AI.OpenAI.Tests.Integration/APIs/AIModels/AIModelsApiTests.cs:19]: Here's what it looks like in github secrets:

image

Comment (Gummimundur) [reply Standard.AI.OpenAI.Tests.Integration/APIs/AIModels/AIModelsApiTests.cs:19]: @hassanhabib Oops this is for my local configuration. Accidentally pushed it. Will fix it

**Pull request merged**